docs/getting-started/gs-credentials.md

```diff
@@ -42,6 +42,7 @@ Notes:
 If you're using a client-server or server-side implementation (see [UID2 Client-Server Integration Guide for Prebid.js](../guides/integration-prebid-client-server.md) or [Client-Server Integration Guide for JavaScript](../guides/integration-javascript-client-server.md)), the API key and client secret allow you to connect to the <Link href="../ref-info/glossary-uid#gl-operator-service">Operator Service</Link> and call API endpoints. These values identify you to the service.
 
 Here is some information about API keys and client secrets:
+- It's important that you keep these values secure. For details, see [Security of API Key and Client Secret](#security-of-api-key-and-client-secret).
```

Comment:
I'd recommend using the word "must" instead of important - we want to make it clear that keeping the values secret is a requirement, rather than a strong suggestion.

Reply:
Done, thx.